e2e/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../tsconfig.base.json",
+  "compilerOptions": {
+    "types": ["cypress", "@testing-library/cypress"],
+    "paths": {
+      "e2e/support/*": ["./support/*"],
```

Comment:
Why not `e2e/*`?
Was there a specific reason to scope it down to the support dir only?

Reply:
When I need to import stuff from the main app, the following are the result from using VS Code's auto-import.

#### With `"e2e/support/*": ["./support/*"]`
<img width="441" alt="image" src="https://user-images.githubusercontent.com/1937582/222179818-cc3ec17f-824b-4701-9d53-77fa59a0600d.png">

#### WIth `"e2e/*": ["./*"]`
<img width="597" alt="image" src="https://user-images.githubusercontent.com/1937582/222180038-c37ef5c8-e63f-4c29-917d-842209fb17f3.png">

I mean they're actually equivalent, but I'll be cursed by all my teammate.

Reply:
Fixed by swapping the position of the `*` and `e2e/*`, so TypeScript tries to resolve the app import using the `*` alias first.

This help not hard-coding `e2e/support/*`, so if we need to refer to any directory under `e2e` in the future, it should work.